packages/valory/protocols/ledger_api/README.md

```diff
@@ -23,6 +23,7 @@ speech_acts:
     terms: ct:Terms
   send_signed_transaction:
     signed_transaction: ct:SignedTransaction
+    kwargs: ct:Kwargs
```

Comment:
this is what i imagined in the begging. However, one thing that worries me is that this is introducing breaking changes, no?